public_dropin_environments/r_lang/dr_requirements.txt

```diff
@@ -2,4 +2,4 @@ numpy==1.19.5
 pandas<=1.5.3
 rpy2==3.5.8
 pyarrow>=0.14.1,<=10.0.1
-datarobot-drum[R]==1.10.2
```

Comment:
what is the change here?
Check the failure: https://jenkins1.devinfra.drdev.io/job/MLOps/job/datarobot-user-models/job/PR-765/4/artifact/test_integration_r_environment.log.html

Reply:
I was using some dev versions for testing and one got commited.  This reverted back to the original, not sure why it shows up as a change.

Reply:
New line was removed. Added a commit. Running tests from the beginning.